packages/protocol/contracts/libs/LibDeploy.sol

```diff
@@ -37,4 +39,35 @@ library LibDeploy {
             OwnableUpgradeable(proxy).transferOwnership(owner);
         }
     }
+
+    function deployERC1967Proxy(
+        address impl,
+        address owner,
+        bytes memory data,
+        TimelockControllerUpgradeable timelock
+    )
+        internal
+        returns (address proxy)
+    {
+        proxy = deployERC1967Proxy(impl, owner, data);
+        if (
+            address(timelock) != address(0) && owner != OwnableUpgradeable(proxy).owner()
```

Comment:
we don't need to call `OwnableUpgradeable(proxy).transferOwnershp(..)`?

Reply:
Its in the previous line: `proxy = deployERC1967Proxy(impl, owner, data);`